### PR TITLE
Move docs/requirments.txt to optional category

### LINKED ISSUE #572: ReadTheDocs should support `pyproject.toml`
This is a substitute for: https://github.com/wemake-services/wemake-django-template/issues/133
Original issue about `Pipfile`: https://github.com/rtfd/readthedocs.org/issues/3181
Original issue about `poetry`: https://github.com/rtfd/readthedocs.org/issues/4912

Currently we have to keep additional `requirements.txt` file, because readthedocs cannot use `pyproject.toml` and `poetry`.

### PR DESCRIPTION
**Pull request by PerchunPak**
Temporairly fixes #572.

### CODE REVIEW

**Comment by PerchunPak** [comment]
Problem in documentation build is that your documentation in template, and template can't be installed because there is no folder `{{ cookiecutter.project_name }}` in it.
Why you need hold your main configuration in template folder? Why not in your `/` folder?

**Comment by sobolevn** [comment]
> Why you need hold your main configuration in template folder? Why not in your / folder?

Because it is a part of the template.
Some people use rtd with our template.

**Comment by sobolevn** [comment]
@PerchunPak will the same `.` configuration work here?

File: .readthedocs.yml [hunk 1]
```diff
@@ -7,7 +7,10 @@ build:
 
 python:
   install:
-    - requirements: '{{cookiecutter.project_name}}/docs/requirements.txt'
+    - method: pip
+      path: '{{cookiecutter.project_name}}'
```
Comment (sobolevn) [inline .readthedocs.yml:11]:
```suggestion
      path: '.'
```

**Comment by PerchunPak** [comment]
> @PerchunPak will the same `.` configuration work here?

No, because docs requirements in `/{{cookiecutter.project_name}}/pyproject.toml`, not in `/pyproject.toml`.
Also it not work with `/{{cookiecutter.project_name}}` because poetry search folder with name `{{ cookiecutter.project_name }}` **IN** `{{cookiecutter.project_name}}` for building wheels. It can be fixed if `docs` folder will move out from `{{cookiecutter.project_name}}`. 
and (optionally) copy `/docs` to `/{{cookiecutter.project_name}}/docs` when generating template. Actually it is idea.

**Pull request closed without merging**